src/smallMultiples.js

```diff
@@ -68,7 +68,7 @@ export default function smallMultiples(id, chart) {
         rangePolygon(this);
 
         // axis tweaks
-        var units = this.current_data[0].values.raw[0][chart.config.unit_col];
+        var units = this.raw_data[0][chart.config.unit_col];
```

Comment:
@jwildfire `this.raw_data` in this case refers to an individual multiple's data, which is already filtered on a single measure.  The reason I switched from `this.current_data` eludes me but it is simpler to pull a datum out of `this.raw_data`.